### PR TITLE
OPSEXP-1756 Switch to nexus for libreoffice packages

### PR DESCRIPTION
**Pull request by gionn**
OPSEXP-1756

### CODE REVIEW

File: roles/transformers/vars/main.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 # vars file for transformers
-libreoffice_repo: "http://downloadarchive.documentfoundation.org/libreoffice/old"
-libreoffice_repo_path: "{{ dependencies_version.libreoffice }}/{{ pkg_format }}/{{ ansible_architecture }}"
-libreoffice_archive_name: /LibreOffice_{{ dependencies_version.libreoffice }}_{{ ansible_system }}_{{ ansible_architecture | regex_replace('_','-') }}_{{ pkg_format }}.tar.gz
+libreoffice_repo: "{{ artifacts_repositories.community.base_url }}/repositories/thirdparty/org/libreoffice/libreoffice-dist"
+libreoffice_repo_path: "{{ dependencies_version.libreoffice }}"
+libreoffice_archive_name: "libreoffice-dist-{{ dependencies_version.libreoffice }}-{{ pkg_format }}.gz"
 libreoffice_archive_url: "{{ libreoffice_repo }}/{{ libreoffice_repo_path }}/{{ libreoffice_archive_name }}"
 libreoffice_archive_sha1_checksum_url: "{{ libreoffice_archive_url }}.sha1"
```
Comment (alxgomz) [inline roles/transformers/vars/main.yml:7]: You don't this anymore then

**Review by alxgomz — CHANGES_REQUESTED** [review]

File: roles/transformers/vars/main.yml (hunk 1 above)
Comment (gionn) [reply roles/transformers/vars/main.yml:7]: nexus still exposing it but in the format ansible expect, and for this reason I had to change https://github.com/Alfresco/alfresco-ansible-deployment/pull/463/files#diff-c4d0ad07c9ccf6e5893488fbf597089dc261c1aea39bfb3e0c69ebb7589d755fR14

Comment (alxgomz) [reply roles/transformers/vars/main.yml:7]: sorry I forgot a word: "You don't NEED this anymore"
I meant this var is not used anywhere now

Comment (gionn) [reply roles/transformers/vars/main.yml:7]: yes [it is](https://github.com/Alfresco/alfresco-ansible-deployment/pull/463/files#diff-c4d0ad07c9ccf6e5893488fbf597089dc261c1aea39bfb3e0c69ebb7589d755fR14)

**Review by alxgomz — APPROVED** [review]

**Pull request merged**